edf/FindForm.cs

```diff
@@ -4,15 +4,24 @@
 using System.IO;
 using System.Reflection;
 using System.Windows.Forms;
+using System.Text.Json;
 
 
 namespace SubReal.EasyDuplicatesFinder
 {
     public partial class FindForm : Form
     {
-        private EdfFiles _edfFiles;
+        private EdfFiles _edfFiles;     
+        
+        const string optionsFileName = "optionsefd.json";
+        public class OptionsEDF
```

Comment:
@Rewalon, Why are you using a nested class?

Reply:
it is not used anywhere else